src/styles/index.scss

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  $minutes-diraction: 3600;
+  $seconds-diraction: 60s;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted black;
+  border-radius: 50%;
+
+  &--speed-up {
+    $minutes-diraction: 600s;
+    $seconds-diraction: 10s;
+  }
+
+  &__circle {
+    position: absolute;
+    width: 5vmin;
+    height: 5vmin;
+    border-radius: 50%;
+    background: #f6a603;
+  }
+
+  &__seconds {
+    transform-origin: bottom;
+    width: 1.5vmin;
+    height: 38vmin;
+    background: #2c8000;
+
+    transform: rotate(0);
+
+    animation-name: handmove;
+    animation-duration: $seconds-diraction;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+
+    &--position {
+      position: absolute;
+      bottom: 40vmin;
+    }
+  }
+
+  &__minutes {
+    transform-origin: bottom;
+    width: 3vmin;
+    height: 20vmin;
+    background: #0700ff;
+
+    transform: rotate(0);
+
+    animation-name: handmove;
+    animation-duration: $minutes-diraction;
+    animation-timing-function: steps(60);
```

Comment:
The `animation-timing-function` property with a value of `steps(60)` for minutes hand is incorrect. It should be `steps(60, end)` to ensure that it stays on each minute mark without any intermediate transition since there are 60 steps in a full rotation for a minute hand.